invite/settings.py

```diff
@@ -36,4 +36,4 @@ def get_service_name():
     return getattr(
         settings,
         'INVITE_SERVICE_NAME',
-        Site.objects.get_current().name)
+        Site.objects.get_current(request=request).name)
```

Comment:
Well, removing the sites framework altogether is an interesting idea. Let me do some digging around to see what it would take to ditch it entirely.